bitcoin/src/lib.rs

```diff
@@ -27,6 +27,7 @@
 #![cfg_attr(all(not(feature = "std"), not(test)), no_std)]
 // Experimental features we need.
 #![cfg_attr(docsrs, feature(doc_auto_cfg))]
+#![cfg_attr(docsrs, feature(doc_notable_trait))]
```

Comment:
```suggestion
#![feature(doc_notable_trait)]
```
Changing to this fixes the error and the docs now show the i in a circle, e.g. below:
![image](https://github.com/user-attachments/assets/0a56a93d-82b4-4299-8325-da17a0ee389c)

Reply:
What happens when you compile with stable (or MSRV) with this?

I thought that you couldn't use `#[feature]` on stable compilers.

Reply:
docs.rs is supposed to set a cfg flag so that we don't need to mess up our codebase for them.

We have lots of `cfg_attr(docsrs, X)` throughout the codebase which seem to work.

Reply:
So if I understand correctly if the `docsrs` flag is set the compiler is always nightly?

Reply:
Yeah. The flag is set by docs.rs, which evidently always uses a nightly compiler.